spec/grape-apiary/route_spec.rb

```diff
@@ -7,15 +7,70 @@
 
   subject { routes.first }
 
-  it 'adds a name helper to routes' do
-    expect(subject.route_name).to eq('widgets')
+  describe '#route_name' do
+    context 'when the route is GET /widgets' do
+      it "is 'widgets'" do
+        expect(subject.route_name).to eq('widgets')
+      end
+    end
   end
 
-  it 'adds a path helper without format' do
-    expect(subject.path_without_format).to eq('/widgets')
+  describe '#api_blueprintified_path' do
+    context 'when the route is GET /widgets/:id' do
+      subject(:individual_widget) { routes.second }
```

Comment:
Is there a little less brittle/more obvious way for you to test the different route types ?  `routes.#{number}` isn't super discoverable 😕

Reply:
Sadly, no... this is how the whole gem works, as in it loads in all the routes and then operates on each one. If you can think of a better way halp plz open a pull request.